### PR TITLE
chore: pkg.pr.new

### PR DESCRIPTION
**Pull request by Aslemammad**
Adding pkg.pr.new jotai! 

This also migrates all the examples to vite.

### CODE REVIEW

**Review by dai-shi — COMMENTED** [review]
Thanks for working on it!

File: examples/hacker_news/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
[the remark this answers is not in the document]
Comment (Aslemammad) [reply examples/hacker_news/.gitignore:2]: i think we need it for node_modules!

[the remark this answers is not in the document]
Comment (dai-shi) [reply examples/hacker_news/.gitignore:2]: It's already ignored in the top-level ignore file.
I think we can remove examples/*/.gitignore.
If there's a problem, we can put them back.

[the remark this answers is not in the document]
Comment (Aslemammad) [reply examples/hacker_news/.gitignore]: done!

File: examples/hacker_news/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="theme-color" content="#000000" />
+    <title>Jotai Examples | Hacker News</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
+  </body>
+</html>
```
Comment (dai-shi) [inline examples/hacker_news/index.html:13]: Missing new line.

File: examples/hacker_news/vite.config.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()]
+})
```
Comment (dai-shi) [inline examples/hacker_news/vite.config.ts:6]: Missing new line.

File: examples/hello/index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      rel="stylesheet"
+    />
+    <meta name="description" content="jotai examples" />
+    <title>Jotai Examples | Hello</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
+  </body>
+</html>
```
Comment (dai-shi) [inline examples/hello/index.html:18]: Missing new line.

File: examples/hello/vite.config.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()]
+})
```
Comment (dai-shi) [inline examples/hello/vite.config.ts:6]: Missing new line.

File: examples/mega-form/index.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Jotai Examples | Mega Form</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
+  </body>
+</html>
```
Comment (dai-shi) [inline examples/mega-form/index.html:13]: Missing new line.

File: examples/mega-form/vite.config.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()]
+})
```
Comment (dai-shi) [inline examples/mega-form/vite.config.ts:6]: Missing new line.

File: examples/text_length/index.html [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+        <a href="https://github.com/react-spring/jotai" target="_blank">GitHub</a>
+      </div>
+      <div class="item-logo"><img src="/cover.svg" alt="Logo" /></div>
+      <div class="item-demo" id="root"></div>
+      <div class="item-code"><img src="/snippet.png" alt="Code" /></div>
+      <div class="item-right"><img src="/castle.jpg" alt="Castle" /></div>
+    </div>
+    <script type="module" src="/src/index.tsx"></script>
+  </body>
+</html>
```
Comment (dai-shi) [inline examples/text_length/index.html:34]: Missing new line.

File: examples/text_length/vite.config.ts [hunk 9]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()]
+})
```
Comment (dai-shi) [inline examples/text_length/vite.config.ts:6]: Missing new line.

File: examples/todos/index.html [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="theme-color" content="#000000" />
+    <title>Jotai Examples | Todos</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
+  </body>
+</html>
```
Comment (dai-shi) [inline examples/todos/index.html:13]: Missing new line.

File: examples/todos/vite.config.ts [hunk 11]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()]
+})
```
Comment (dai-shi) [inline examples/todos/vite.config.ts:6]: Missing new line.

File: examples/todos_with_atomFamily/index.html [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="theme-color" content="#000000" />
+    <title>Jotai Examples | Todos with atomFamily</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
+  </body>
+</html>
```
Comment (dai-shi) [inline examples/todos_with_atomFamily/index.html:13]: Missing new line.

File: examples/todos_with_atomFamily/vite.config.ts [hunk 13]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()]
+})
```
Comment (dai-shi) [inline examples/todos_with_atomFamily/vite.config.ts:6]: Missing new line.

**Review by dai-shi — CHANGES_REQUESTED** [review]
😄

**Pull request merged**